docs/DEVELOPING.md

```diff
@@ -343,6 +343,114 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## Microsoft SQL Connector
+
+The [Elastic Microsoft SQL connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/mssql.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+**Prerequisite**
+
+The Microsoft SQL connector requires an ODBC driver to execute. To install ODBC driver please refer following [documentation](https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-driver-manager).
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Microsoft SQL details](#gather-microsoft-sql-details)
+2. [Configure Microsoft SQL connector](#configure-microsoft-sql-connector)
+
+#### Gather Microsoft SQL details
+
+Collect the information that is required to connect to your Microsoft SQL Server:
+
+- The server host address where the Microsoft SQL Server is hosted.
+- The port where the Microsoft SQL server is hosted.
+- The username the connector will use to log in to Microsoft SQL.
+- The password the connector will use to log in to Microsoft SQL.
+- The Microsoft SQL database name where the connector will crawl to fetch rows.
+
+#### Configure Microsoft SQL connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `host`
+
+The server host address where the Microsoft SQL Server is hosted. Default value is `127.0.0.1`. Examples:
+
+  - `192.158.1.38`
+  - `demo.instance.demo-region.demo.service.com`
+
+##### `port`
+
+The server port where Microsoft SQL Server is available. Default value is `9090`. Examples:
+
+  - `5432`
+  - `9090`
+
+##### `user`
+
+The username of the account for Microsoft SQL Server. Default value is `admin`. Examples:
+
+##### `password`
+
+The password of the account to be used for the Microsoft SQL Server. Default value is `Password_123`.
+
+##### `database`
+
+Name of the Microsoft SQL Server database. Default value is `xe`. Examples:
+
+  - `employee_database`
+  - `customer_database`
+
+##### `tables`
+
+Comma-separated list of tables. For * in string connector will fetch data for all the Tables. Default value is `*`. Examples:
```

Comment:
```suggestion
Comma-separated list of tables. Default value is `*`, which indicates the connector will fetch data for all the tables. Examples:
```

Reply:
To make it generic as other dialects  change to `For * in string connector will fetch data for all the Tables`.

Reply:
Or we can change it to description (as in another PR)

https://github.com/elastic/connectors-python/blob/ae39452db0a70dc2e677d251b4d266e43bdc0513/docs/DEVELOPING.md?plain=1#L413

I feel it's easier to understand than `For * in string connector will fetch data for all the Tables`

Reply:
Yes, I have changed it `The Microsoft SQL connector will fetch data from all tables present in the configured database if the value is * .` [here](https://github.com/elastic/connectors-python/pull/557/files#diff-f8ff34865ce88864196f2848aebde961f27a4c1e8ce77313a4254dd948d32cfbR414) in same commit.